contracts/src/CrossChainApplications/ERC20Bridge/ERC20Bridge.sol

```diff
@@ -73,15 +73,29 @@ contract ERC20Bridge is IERC20Bridge, ITeleporterReceiver, ReentrancyGuard {
     uint256 public constant MINT_BRIDGE_TOKENS_REQUIRED_GAS = 200_000;
     uint256 public constant TRANSFER_BRIDGE_TOKENS_REQUIRED_GAS = 300_000;
 
+    // Errors
+    error InvalidTeleporterMessengerAddress();
+    error BridgeTokenWithinSameChain();
```

Comment:
Nit: `BridgeTokenWithinSameChain` and `CannotBridgeWrappedToken` seems like inconsistent naming. I think we should probably have "cannot for both of them.